src/main/java/theChillys/chillys_radio/station/repository/IStationRepository.java

```diff
@@ -12,6 +12,6 @@ public interface IStationRepository extends JpaRepository<Station, Long> {
     List<Station> findAllByOrderByClickcountDesc();
     List<Station> findAllByOrderByVotesDesc();
     Optional<Station> findByStationuuid(String stationuuid);
-    List<Station> findStationByTagsOrCountryOrLanguage(String tags,String country, String language);
+    List<Station> findStationByGenreOrCountryOrLanguage(String tags, String country, String language, String s);
```

Comment:
Wrong method. Fiels Genre is not present in repo.
Variables order differs from Service method, also naming is not understandable